gitlab/base.py

```diff
@@ -45,13 +45,19 @@ class RESTObject(object):
 
     _id_attr: Optional[str] = "id"
     _attrs: Dict[str, Any]
+    _list_created_object: bool  # Indicates if object was created from a list() action
```

Comment:
just a nitpick: to be concise (and similar to other existing params like `_from_parent_attrs`) would this also work? It would seem quite clear to me already.

```suggestion
    _from_list: bool
```

Reply:
I'm not sure. I feel like `_list_created_object` reduces the cognitive load for readers of the code more than `_from_list`. Also I like the comment to help people understand it, like me two months from now 😏

Reply:
Since you said it was a "nitpick" I left it as is. If you feel more strongly than that I can change it.

Reply:
Ok :) Since we are already in `RESTObject` that's why I was thinking it was clear. Maybe `_created_from_list` would be somewhere in the middle